src/dmd/cppmangle.d

```diff
@@ -935,16 +974,30 @@ private final class CppMangleVisitor : Visitor
             source_name(d, true);
             buf.writeByte('E');
         }
-        //char beta[6] should mangle as "beta"
+        else if (isNested)
+        {
+            buf.writestring("_Z");
+            source_name(d, false);
+        }
         else
         {
-            if (!isNested)
-                buf.writestring(d.ident.toString());
-            else
+            auto varTags = ABITagContainer.forSymbol(d);
+            if (varTags)
```

Comment:
why not just
```
if (ABITagContainer.forSymbol(d))
```

Reply:
Indeed, this is a leftover of a previous iteration, will fix.